Packs/EclecticIQ/Integrations/EclecticIQ/EclecticIQ.py

```diff
@@ -1,741 +1,676 @@
-
-    original_extract_response = original_extract(observable_id)
-
-    response = dict(processed_extract_response)
-    response['data'].extend(original_extract_response['data'])
-    response['total_count'] += original_extract_response['total_count']
-
-    if 'total_count' in response and response['total_count'] == 0:
-        demisto.results('No results found')
-        return
-
-    context_outputs = []
-    human_readable = ''
-
-    entities = response.get('data')
-
-    for entity in entities:  # type: ignore
-
-        entity_data = entity.get('data', {})
-        test_mechanisms = entity_data.get('test_mechanisms', {})
-        entity_meta = entity.get('meta', {})
-
-        context_output = {
-            'Title': entity_data.get('title'),
-            'ID': entity.get('id'),
-            'Analysis': entity_data.get('description'),
-            'EstimatedStartTime': entity_meta.get('estimated_threat_start_time'),
-            'EstimatedObservedTime': entity_meta.get('estimated_observed_time'),
-            'HalfLife': entity_meta.get('half_life')
-        }
-
-        if context_output['Analysis']:
-            # Removing unnecessary whitespaces from the string
-            context_output['Analysis'] = ' '.join(context_output['Analysis'].split())
-
-        if context_output['HalfLife']:
-            # API returns a number, we add the time format to it
-            context_output['HalfLife'] = str(context_output['HalfLife']) + ' Days'
-
-        human_readable += tableToMarkdown(f'Observable ID {observable_id} related entities', context_output)
-
-        test_mechanisms_output = []
-
-        for mechanism in test_mechanisms:
-
-            mechanism_output = {
-                'Type': mechanism.get('test_mechanism_type')
-            }
-
-            mechanism_rules = mechanism.get('rules')
-
-            mechanism_rules_outputs = []
-
-            for rule in mechanism_rules:
-
-                mechanism_rules_outputs.append(rule.get('value'))
-
-            mechanism_output['Rule'] = mechanism_rules_outputs
-
-            test_mechanisms_output.append(mechanism_output)
-
-        if test_mechanisms_output:
+    api_key = demisto.params().get('apikey')
 
-            context_output['TestMechanism'] = test_mechanisms_output
-            human_readable += tableToMarkdown('Test mechanisms', test_mechanisms_output, removeNull=True)
+    # get the service API url
+    base_url = demisto.params().get('url')
 
-        sources = entity.get('sources')
+    # if your Client class inherits from BaseClient, SSL verification is
+    # handled out of the box by it, just pass ``verify_certificate`` to
+    # the Client constructor
+    verify_certificate = not demisto.params().get('insecure', False)
 
-        sources_output = []
+    # if your Client class inherits from BaseClient, system proxy is handled
+    # out of the box by it, just pass ``proxy`` to the Client constructor
```

Comment:
Please remove the developer help comments.

Reply:
Done